server/database/pg/index.js

```diff
@@ -1,12 +1,11 @@
-require('dotenv').config();
 const { Pool } = require('pg');
 const path = require('path');
 
 const pool = new Pool({
-  user: process.env.USER,
-  host: process.env.HOST,
-  database: process.env.DATABASE,
-  port: process.env.PORT,
+  user: 'ericsong',
```

Comment:
Any particular reason you hard coded this information instead of adding it to the .ENV file?

Reply:
I tried hardcoding this part to eliminate potential sources of error during my debugging process. But thanks for the reminder - I'll make sure to add the dotenv configuration before merging.